app/views/nominations/request_nomination_invite/success.html.erb

```diff
@@ -5,9 +5,10 @@
 
     <%= govuk_panel title_text: "Your school has been sent a link", text: nil, classes: "govuk-!-margin-bottom-7" %>
 
-    <p class="govuk-body">We’ve sent a link to the school’s email address at</p>
-
-    <p><strong><%= EmailDecorator::new(@school_email).partial_email %></strong></p>
+    <p class="govuk-body">
+      We’ve sent an email to
+      <strong><%= EmailDecorator::new(@school_email).partial_email %></strong>
+    </p>
 
     <h2 class="govuk-heading-m">What happens next</h2>
```

Comment:
We’ve updated this content, so it's no longer a heading and a list. See [content in the prototype](https://manage-training-for-ects-5c9ed99fc0a8.herokuapp.com/check-school-email).